rancher2/config.go

```diff
@@ -88,6 +88,28 @@ func (c *Config) getK8SVersion() (string, error) {
 	return c.K8SVer, nil
 }
 
+func (c *Config) IsRancherVersionLessThanl(ver string) (bool, error) {
```

Comment:
is this function used? and the "l" at the end?

Reply:
Fixed function name